source/overview/corba.rst

```diff
@@ -0,0 +1,32 @@
+10 things you should know about CORBA
+=====================================
+
+1.  ***You don’t need to know CORBA to work with TANGO***
```

Comment:
No no Piotr, don't worry! I'll do it! It is the best to learn how to do it... ;)